src/adapters.coffee

```diff
@@ -177,7 +177,7 @@ class Adapters
       utils.sha1("#{@namespace}.#{experiment.name}.#{experiment.user_id}")
 
     _track: (experiment, variant, goal) ->
-      utils.log("Persistent Queue Keen track: #{experiment.name}, #{variant}, #{event}")
+      utils.log("Persistent Queue Keen track: #{experiment.name}, #{variant}, #{goal.name}")
```

Comment:
actually I think we can remove the keen adapter. If anyone uses it, they can use their own adapter!

Reply:
Do you want me to remove it?

Reply:
we can do it separately. just thought it’s something that can make Alephbet smaller